rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -445,10 +427,10 @@ public Void doInTransaction(Session session) {
                     }
                 });
             } catch (RuntimeException e) {
-                throw new RuntimeException("Exception occurred while removing node " + rmNodeData.getName(), e);
+                throw new RuntimeException("Exception occurred while removing node " + rmNodeData.getNodeUrl(), e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)